src/Packetery/Module/Carrier/CountryListingPage.php

```diff
@@ -235,6 +263,7 @@ private function getActiveCountries(): array {
 					get_admin_url( null, 'admin.php' )
 				),
 				'activeCarriers'            => $activeCarriers,
+				'allCarriers'               => $allCarriers,
```

Comment:
activeCarriers potřebuješ jen v případě původní šablony,  allCarriers zase potřebuješ jen v případě nové šablony.  Vracel bych tedy jen ty data, které opravdu potřebuješ. To můžeš zajistit parametrem v metodět getActiveCountries()

Reply:
Upraveno.